### DISCUSSION TITLE
Melhoras práticas sobre responsividade

### DISCUSSION THREAD

**Discussion by piresmarina** [balta-io/forum#700] [Tira Dúvidas] [answered]
Estou executando o passo a passo do projeto (fiz até Estilizando a segunda section). Observei que:
1. a imagem na primeira section fica muito maior do que aparece no vídeo do curso (isto foi comprovado porque na criação da segunda section, foi aplicado uma width de 80% na imagem para aumentá-la e quando apliquei esta estilização a imagem diminuiu).
![tamanho-imagem-primeira-section](https://github.com/user-attachments/assets/ba326073-082b-4285-8ef1-9c396df03af0)
3. a segunda section sobrepõe a primeira section. No curso não há esta sobreposição. E, até o momento, não foi adicionada nenhuma margin para separar as seções.
![sobreposição-sections](https://github.com/user-attachments/assets/d41d18d9-7b42-43e3-8c64-abc4510cc4c8)

**Comment by henriquegiaretta** [comment, marked as answer]
Olá @piresmarina, como estás? Tudo bem por aí.

Vou tentar te ajudar aqui com os tópicos.

Sobre o cenário 1: você pode aplicar um **width** ou **height** sem problemas nenhum. Essa diferença de tamanho é devido aos diferentes tipos de resoluções. Provavelmente a resolução de monitor utilizada no curso está diferente da sua. Como não utilizamos **svg** (que seria possível redimensionar a imagem) devemos fazer essas alterações manualmente, com **width** e **height**.

Sobre o cenário 2: o motivo disso acontecer é que estamos utilizando 100% do view height (vh) da tela. Como a altura da tela está reduzida, ele tenta compensar para utilizar os 100% (de espaço disponível da tela) para colocara a imagem. Veja abaixo:

![image](https://github.com/user-attachments/assets/d8ab314d-058a-45da-9205-bd29e37ebe15)

Agora, se não utilizarmos o **vh** conseguimos atingir o resultado esperado.

![image](https://github.com/user-attachments/assets/6bf0c74f-14be-4ff7-955e-0dd792abe7d4)

Ambas estão dentro da classe **container** e você pode alternar entre essas configurações a vontade.

Espero que eu tenha te ajudado. Qualquer dúvida é só colocar aqui :)
